### DISCUSSION TITLE
How to use pager with query results

### DISCUSSION THREAD

**Discussion by juergen-de** [OrchardCMS/OrchardCore#11774] [Q&A]
Hi, any idea how the pager can be used with a liquid variable ?

{% assign timelineItems = Queries.BlogPostByTimeline | query %}
    {% for item in timelineItems %}
    {{ item | shape_build_display: "Summary" | shape_render }}
{% endfor %}

The existing pager does nothing of course but I do not really see how I can do paging here. Any ideas ?

**Comment by Skrypt** [comment]
See https://github.com/OrchardCMS/OrchardCore/issues/4140
